contracts/core/LensHub.sol

```diff
@@ -576,13 +589,19 @@ contract LensHub is ILensHub, LensNFTBase, VersionedInitializable, LensMultiStat
     function collect(
         uint256 profileId,
         uint256 pubId,
-        bytes calldata data
+        bytes calldata data,
+        uint256 followedFromProfileId
```

Comment:
Would move it up, before the `data` param, so its in the same order as the `follow` params